lxd/storage_pools_config.go

```diff
@@ -50,8 +50,6 @@ func storagePoolValidateConfig(name string, driver string, config map[string]str
 	if config["source"] == "" {
 		if driver == "dir" {
 			config["source"] = filepath.Join(shared.VarPath("storage-pools"), name)
-		} else if driver != "lvm" {
-			config["source"] = filepath.Join(shared.VarPath("disks"), name)
 		}
 	}
```

Comment:
My branch makes sure that storagePoolValidateConfig doesn't call storagePoolFillDefault as it's otherwise breaking pool configuration when LXD decides to fill in read-only properties